src/utils/source-maps.js

```diff
@@ -94,7 +94,7 @@ export async function mapLocation(
     return getGeneratedLocation(state, source, location, sourceMaps);
   }
 
-  return sourceMaps.getOriginalLocation(location, source);
```

Comment:
Not sure why `source` was passed here but seems invalid given the signature of `getOriginalLocation`.

Reply:
looks like a good catch